src/navier_stokes/INSCollocatedVelocityBcCoef.cpp

```diff
@@ -53,8 +54,6 @@ namespace hier
 {
 template <int DIM>
 class Variable;
-template <int DIM>
-class Index;
```

Comment:
Should we remove all these forward declarations? They are used inconsistently throughout the library.